src/Views/ValorantTeam.jsx

```diff
@@ -31,19 +32,19 @@ const ValorantTeam = () => {
       </div>
       <div className="l-container">
         {/* FOR MAP */}
-        {/* {teams.map((team) => (
-          <div className="l-container" key={team.teamLogo}>
+        {teams.map((team) => (
+          <div className="l-container" key={team.teamLogoURL}>
             <Link to="/team">
-              <Astrateamcards teamlogo={team.teamLogo} tname="Millennia" />
+              <Astrateamcards teamlogo={team.teamLogoURL} tname="Millennia" />
             </Link>
           </div>
-        ))} */}
+        ))}
 
         {/* FOR UI OUTPUT REFERENCE ONLY */}
-        <Astrateamcards teamlogo={sol} tname="Millennia" />
-        <Astrateamcards teamlogo={millen} tname="Millennia" />
+        {/* <Astrateamcards teamlogo={teams[0].teamLogoURL} tname="Millennia" /> */}
+        {/* <Astrateamcards teamlogo={millen} tname="Millennia" />
         <Astrateamcards teamlogo={hlx} tname="Millennia" />
-        <Astrateamcards teamlogo={prs} tname="Millennia" />
+        <Astrateamcards teamlogo={prs} tname="Millennia" /> */}
```

Comment:
We can now remove this before moving to the main file. Lets avoid leaving traces log.